support/map.h

```diff
@@ -2014,13 +2014,14 @@ Mono_Posix_ToUtimbuf (struct utimbuf *from, struct Mono_Posix_Utimbuf* to);
 
 
 struct Mono_Unix_UnixSignal_SignalInfo {
-	int   signum;
-	int   count;
+	int   signum;   // Read by signal handlers
```

Comment:
These comments are pointless; `map.h` is a generated file (see comment at top of file), and thus the comments will be lost the next time `create-native-map` is run.

If these comments are needed, they should be placed in the C# declaration.